### ISSUE TITLE
Unable to edit tags

### ISSUE THREAD

**Issue by adamcaudill** [paragonie/airship#144]
### Summary

When editing a tag, an error is displayed: "Please match the requested format"

### Expected Outcome

Tag is updated.

### What Actually Happened

Error occurs, and it is not possible to update tag.

**Comment by paragonie-scott** [comment]
I couldn't reproduce a failure on this one. If you can collect any more data about the specific error that occurred, I'd greatly appreciate it.

**Comment by adamcaudill** [comment]
Based on how it's acting, looks like it's happening on the client side - doesn't appear to hit the server at all. While looking at it, noticed the following error in the JS console (appears as soon as the textbox gets focus):

```
Pattern attribute value ^[^\#]+$ is not a valid regular expression: Uncaught SyntaxError: Invalid regular expression: /^[^\#]+$/: Invalid escape
```

I'm too tired to dig into it anymore tonight, so thought I would share what little I found.

**Comment by kelunik** [comment]
It says `Invalid escape`, so probably must be `/^[^\\#]+$/` instead.

**Comment by paragonie-scott** [comment]
Okay, that makes sense. I'll look for more patterns that might be broken and fix them before I tag v1.3.1.
